src/ros_vrpn_client.cpp

```diff
@@ -263,7 +263,8 @@ void inline getTimeStamp(const ros::Time& vicon_stamp, ros::Time* timestamp) {
     }
     case kTranslation: {
       if(device_time_translator) {
-        *timestamp = device_time_translator->update(vicon_stamp.toNSec(), ros::Time::now(), 0.0);
+	// Compensate for latency according to https://core.ac.uk/download/pdf/211853264.pdf
+        *timestamp = device_time_translator->update(vicon_stamp.toNSec(), ros::Time::now(), -0.0549);
```

Comment:
Thanks for this. I'll just put this here as a reminder to make it a ROS param, should we decide to merge this in the future :)